masterdata/returnRequest/schema.json

```diff
@@ -0,0 +1,211 @@
+          "type": "string"
+        },
+        "paidWithCc": {
+          "type": "boolean"
+        }
+      }
+    },
+    "items": {
+      "type": "array",
+      "required": true,
+      "items": {
+        "type": "object",
+        "properties": {
+          "orderItemIndex": { "type": "integer", "required": true },
+          "quantity": { "type": "integer", "required": true },
+          "verifiedItems": { "type": "integer" },
+          "reasonCode": { "type": "string", "required": true },
+          "reason": { "type": "string", "required": true },
+          "condition": {
+            "type": "string",
+            "required": true,
+            "enum": [
+              "newWithBox",
+              "newWithoutBox",
+              "usedWithBox",
+              "usedWithoutBox"
+            ]
+          }
+        }
+      }
+    },
+    "dateSubmitted": { "type": "string", "format": "date-time" },
+    "userComment": { "type": "string", "maxLength": 300 },
+    "refundData": {
+      "type": ["object", "null"],
+      "required": true,
+      "properties": {
+        "invoiceNumber": { "type": "string", "required": true },
+        "invoiceValue": { "type": "integer", "required": true },
+        "refundedItemsValue": { "type": "integer", "required": true },
+        "refundedShippingValue": { "type": "integer" },
+        "giftCard": {
+          "type": "object",
+          "properties": {
+            "id": {
+              "type": "string"
+            },
+            "code": {
+              "type": "string"
+            }
+          }
+        },
+        "items": {
+          "type": "array",
+          "required": true,
+          "items": {
+            "type": "object",
+            "properties": {
+              "orderItemIndex": { "type": "integer", "required": true },
+              "quantity": { "type": "integer", "required": true },
+              "restockFee": { "type": "integer" }
+            }
+          }
+        }
+      }
+    },
+    "refundStatusData": {
+      "type": "array",
+      "required": true,
+      "items": {
+        "type": "object",
+        "properties": {
+          "status": { "$ref": "#/$defs/status", "required": true },
+          "submittedBy": { "type": "string" },
+          "comment": {
+            "type": "object",
+            "properties": {
+              "value": { "type": "string" },
+              "visibleForCustomer": { "type": "boolean" }
+            }
+          }
+        }
+      }
+    }
+  },
+  "$defs": {
+    "status": {
+      "type": "string",
+      "enum": [
+        "new",
+        "processing",
+        "pickedUpFromClient",
+        "pendingVerification",
+        "packageVerified",
+        "amountRefunded",
+        "denied"
+      ]
+    }
+  },
+  "v-cache": false,
+  "v-default-fields": [
+    "id",
+    "createdIn",
+    "orderId",
+    "sequenceNumber",
+    "dateSubmitted"
+  ],
+  "v-indexed": ["id", "createdIn", "orderId", "customerProfileData"],
```

Comment:
For now, only indexing the fields I see we can use to fetch the document. We should keep it always as minimum as possible and NEVER delete a property from it to avoid breaking indexation.